config/services.yaml

```diff
@@ -32,20 +42,7 @@ services:
 
     command_handlers:
         namespace: StepupReadId\Application\
-        resource: '%kernel.project_dir%/src/Application/**/*CommandHandler.php'
+        resource: '%kernel.project_dir%/src/Application/**/*Handler.php'
```

Comment:
Whats the reason behind making the command bus shared for commands and queries?

Reply:
Because we don't do real CQRS. It's the same synchronous bus, so there are no reason to complicate the configuration. Here query and command is just semantic. Probably I could change the suffixes to Message and MessageHandler to avoid confusions.